doc/api/globals.md

```diff
@@ -1273,5 +1278,4 @@ A browser-compatible implementation of [`WritableStreamDefaultWriter`][].
 [buffer section]: buffer.md
 [built-in objects]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects
 [timers]: timers.md
-[webassembly-mdn]: https://developer.mozilla.org/en-US/docs/WebAssembly
-[webassembly-org]: https://webassembly.org
+[webassembly-mdn]: https://developer.mozilla.org/en-US/docs/WebAssembly
```

Comment:
These changes seem incorrect.

Reply:
Do we need to keep these two ?

Reply:
They are both still used in the file, so I believe so. In general though, you should avoid touching unrelated code in PRs.

Reply:
My unrelated change fixes didn't resolve that issue, my bad!